bgpd/bgp_attr.c

```diff
@@ -4752,7 +4752,7 @@ void bgp_packet_mpattr_prefix(struct stream *s, afi_t afi, safi_t safi,
 			stream_putl(s, addpath_tx_id);
 		/* Label, RD, Prefix write. */
 		stream_putc(s, p->prefixlen + 88);
-		stream_put(s, label, BGP_LABEL_BYTES);
+		stream_put(s, label, MPLS_LABEL_LEN_BYTES);
 		stream_put(s, prd->val, 8);
```

Comment:
Is it interesting to change `MPLS_LABEL_LEN_BYTES` to `MPLS_LABEL_LEN_BYTES * num_labels` ?

Reply:
This is `SAFI_MPLS_VPN` case - I didn't touch that - my semantic changes are only for `SAFI_LABELED_UNICAST`.
For other SAFI* I've tried my best not to change behaviour and changed something only when I had two.
I had to add `MPLS_LABEL_LEN_BYTES` to use it in `lib/stream.c` and then maintainer suggested just removing duplicate constant `BGP_LABEL_BYTES`, so I did that.